src/multiple_choice/config.py

```diff
@@ -66,13 +75,15 @@ def updateSyncedConfig():
     mw.col.set_config("mc_conf", tmp_conf)
     mw.col.setMod()
```

Comment:
This one is also deprecated.
`setMod() is deprecated: no longer required`
There's another one on line 63.
```suggestion
```